pkg/shared/accountmgmtutil/ams.go

```diff
@@ -86,56 +87,80 @@ func GetOrgQuotas(f *factory.Factory, spec *remote.AmsConfig) (*OrgQuotas, error
 		return nil, err
 	}
 
-	var standardQuotas, marketplaceQuotas, trialQuotas []QuotaSpec
+	var standardQuotas, marketplaceQuotas, trialQuotas, evalQuotas []QuotaSpec
 	for _, quota := range quotaCostGet.GetItems() {
 		quotaResources := quota.GetRelatedResources()
 		for i := range quotaResources {
 			quotaResource := quotaResources[i]
 			if quotaResource.GetResourceName() == spec.ResourceName {
-				if quotaResource.GetProduct() == spec.TrialProductQuotaID {
+				switch quotaResource.GetProduct() {
+				case spec.TrialProductQuotaID:
 					trialQuotas = append(trialQuotas, QuotaSpec{QuotaTrialType, 0, quotaResource.BillingModel, nil})
-				} else if quotaResource.GetProduct() == spec.InstanceQuotaID {
+				case spec.InstanceQuotaID:
 					remainingQuota := int(quota.GetAllowed() - quota.GetConsumed())
 					if quotaResource.BillingModel == QuotaStandardType {
 						standardQuotas = append(standardQuotas, QuotaSpec{QuotaStandardType, remainingQuota, quotaResource.BillingModel, nil})
 					} else if quotaResource.BillingModel == QuotaMarketplaceType {
 						marketplaceQuotas = append(marketplaceQuotas, QuotaSpec{QuotaMarketplaceType, remainingQuota, quotaResource.BillingModel, quota.CloudAccounts})
 					}
+				case "RHOSAKEval":
```

Comment:
Do we need to be using a string literal here

Reply:
These literals are to be fetched from a GitHub file, it will work only after merging this PR, I will do it in a follow up PR.